text/0000-lts.md

```diff
@@ -0,0 +1,164 @@
+- Feature Name: lts
+- Start Date: 2018-06-21
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Add an additional release channel for long term releases.
+
+# Motivation
+[motivation]: #motivation
+
+Rust's current release model has, overall, worked very well for us. We maintain
+a strong degree of backwards compatibility despite producing a new stable
+release very frequently. Relative to many other languages, it is very easy to
+upgrade between Rust stable releases.
+
+However, for various institutional reasons, many users do not find it
+convenient and desirable to upgrade with the frequency that Rust releases. So
```

Comment:
IMO, this should be substantiated by at least some examples of the institutional reasons and the users, and that are specific to Rust.